src/energyplus/Test/SubSurface_GTest.cpp

```diff
@@ -127,3 +127,224 @@ TEST_F(EnergyPlusFixture, ForwardTranslator_SubSurface) {
+  vertices.push_back(Point3d(0, 0, 0));
+  vertices.push_back(Point3d(4, 0, 0));
+  vertices.push_back(Point3d(4, 0, 2));
+  Surface surface(vertices, model);
+  surface.setSpace(space);
+  surface.setSurfaceType("Wall");
+
+  vertices.clear();
+  vertices.push_back(Point3d(0.5, 0, 1.5));
+  vertices.push_back(Point3d(0.5, 0, 0.5));
+  vertices.push_back(Point3d(1.5, 0, 0.5));
+  vertices.push_back(Point3d(1.5, 0, 1.5));
+
+  SubSurface subSurface1(vertices, model);
+  subSurface1.setSurface(surface);
+  subSurface1.setSubSurfaceType("FixedWindow");
+
+  vertices.clear();
+  vertices.push_back(Point3d(2.5, 0, 1.5));
+  vertices.push_back(Point3d(2.5, 0, 0.5));
+  vertices.push_back(Point3d(3.5, 0, 0.5));
+  vertices.push_back(Point3d(3.5, 0, 1.5));
+
+  SubSurface subSurface2(vertices, model);
+  subSurface2.setSurface(surface);
+  subSurface2.setSubSurfaceType("FixedWindow");
+
+  double windowWallRatio = surface.windowToWallRatio();
+  EXPECT_NEAR(windowWallRatio, 0.25, 0.01);
+
+  WindowPropertyFrameAndDivider frame1(model);
+  frame1.setFrameWidth(0.030);
+  subSurface1.setWindowPropertyFrameAndDivider(frame1);
+
+  WindowPropertyFrameAndDivider frame2(model);
+  frame2.setFrameWidth(0.030);
+  subSurface2.setWindowPropertyFrameAndDivider(frame2);
+
+  windowWallRatio = surface.windowToWallRatio();
+  EXPECT_NEAR(windowWallRatio, 0.281, 0.01);
+}
+
+TEST_F(EnergyPlusFixture, Issue_4361_Multi_Subsurfaces_Overlapping) {
+  Model model;
+
+  ThermalZone thermalZone(model);
+
+  Space space(model);
+  space.setThermalZone(thermalZone);
+
+  std::vector<Point3d> vertices;
+  vertices.push_back(Point3d(0, 0, 2));
+  vertices.push_back(Point3d(0, 0, 0));
+  vertices.push_back(Point3d(4, 0, 0));
+  vertices.push_back(Point3d(4, 0, 2));
+  Surface surface(vertices, model);
+  surface.setSpace(space);
+  surface.setSurfaceType("Wall");
+
+  vertices.clear();
+  vertices.push_back(Point3d(0.5, 0, 1.5));
+  vertices.push_back(Point3d(0.5, 0, 0.5));
+  vertices.push_back(Point3d(1.5, 0, 0.5));
+  vertices.push_back(Point3d(1.5, 0, 1.5));
+
+  SubSurface subSurface1(vertices, model);
+  subSurface1.setSurface(surface);
+  subSurface1.setSubSurfaceType("FixedWindow");
+
+  vertices.clear();
+  vertices.push_back(Point3d(1.51, 0, 1.5));
+  vertices.push_back(Point3d(1.51, 0, 0.5));
+  vertices.push_back(Point3d(2.51, 0, 0.5));
+  vertices.push_back(Point3d(2.51, 0, 1.5));
+
+  SubSurface subSurface2(vertices, model);
+  subSurface2.setSurface(surface);
+  subSurface2.setSubSurfaceType("FixedWindow");
+
+  double windowWallRatio = surface.windowToWallRatio();
+  EXPECT_NEAR(windowWallRatio, 0.25, 0.01);
+
+  WindowPropertyFrameAndDivider frame1(model);
+  frame1.setFrameWidth(0.030);
+  subSurface1.setWindowPropertyFrameAndDivider(frame1);
+
+  WindowPropertyFrameAndDivider frame2(model);
+  frame2.setFrameWidth(0.030);
+  subSurface2.setWindowPropertyFrameAndDivider(frame2);
+
+  windowWallRatio = surface.windowToWallRatio();
```

Comment:
You'll want to check the standards to see if you should include frame area and/or divider area in the WWR calculations.